packages/tenant-process/src/model/domain/apiConverter.ts

```diff
@@ -88,6 +88,7 @@ export function toApiTenantAttribute(
       declared: {
         id: attribute.id,
         assignmentTimestamp: attribute.assignmentTimestamp.toJSON(),
+        revocationTimestamp: attribute.revocationTimestamp?.toJSON(),
```

Comment:
Change cherry picked from https://github.com/pagopa/interop-be-monorepo/pull/776/files to make testing work for this PR